website/docs/faqs/Accounts/cloud-upgrade-instructions.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "How to upgrade a dbt Cloud account"
+id: "cloud-upgrade-instructions"
+description: "Instructions for upgrading a dbt Cloud account after the trial ends."
+---
+
+dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud users and explains how to select a plan in order to continue using dbt Cloud. 
+
+:::tip Before you begin
+- You **_must_** be part of the [Owner](/docs/collaborate/manage-access/self-service-permissions) user group to make billing changes. Users not included in this group will not see these options.
+- All amounts shown in dbt Cloud are in U.S. Dollars (USD)   
+:::
+
+
+## Select a plan 
+When your [14 day trial](https://www.getdbt.com/signup/) ends or if your subscription payment is past due , you'll need to select a plan in order to continue using your account:
+
+- Upon logging in, you should see an "Account locked" pop up message with instructions to unlock your account and update your payment details
+- Click **Go to Billing** to go to the billing page
+- Under **Billing**, you can review the available dbt Cloud [plans](https://www.getdbt.com/pricing/) and their features
+
+
+To unlock your account and select a plan, review the following guidance per plan type:
+
+### Developer plan
+
+1. To select a Developer plan, choose the Developer tab under **Billing**. 
+2. Click **Select Plan** on the right. 
+3. Confirm your plan selection on the pop up message. 
+4. This automatically unlocks your dbt Cloud account and you should now enjoy the benefits of the Developer plan. 🎉
+
+<Lightbox src="/img/docs/dbt-cloud/downgrade-dev-flow.gif"/>
+
+### Team plan
+
+1. When your trial expires, your account's default plan enrollment will be a Team plan. 
+2. To unlock your account and continue using the Team plan, you need to enter your payment details. 
+3. Go to **Payment Information** and click **Edit** on the right.
+4. Enter your payment details and click **Save**.
+5.  This automatically unlocks your dbt Cloud account and you should now enjoy the benefits of the Team plan. 🎉
+
+<Lightbox src="/img/docs/dbt-cloud/trial-team-flow.gif"/>
+
+### Enterprise plan
+
+1. If you're interested in our Enterprise plan, select the Enterprise tab under **Billing**.
+2. Click **Contact Sales** on the right. This opens a chat window for you to contact the dbt Cloud Support team, who will connect you to our Sales team.
+3. Once you submit your request, our Sales team will contact you with more information. 
```

Comment:
At this point, the product is locked, and they can't use it, so is there an estimated time we can provide for how long this process will take?

Reply:
good question, i purposely left it out because i don't think we have an estimated time frame for this. i'd say probably a week but not sure if we want to hold ourselves to this. Unless using words like 'immediately contact you...' is a good compromise?

Reply:
It's good to me as is, I was just wondering if there was well defined, but it seems like it could be flexible.